dense_vector/challenges/default.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "index-and-search",
+  "description": "indexes vectors, force merges to a single segment, then searches them",
+  "default": true,
+  "schedule": [
+    {
+      "operation": {
+        "operation-type": "delete-index"
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "create-index"
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "cluster-health",
+        "request-params": {
+          "wait_for_status": "green"
+        },
+        "retry-until-success": true
+      }
+    },
+    {
+      "operation": "index-append",
+      "warmup-time-period": {{ bulk_warmup | default(240) | int }},
+      "clients": {{bulk_indexing_clients | default(8)}}
+    },
+    {
+      "operation": {
+        "operation-type": "force-merge",
+        "max-num-segments": 1,
+        "request-timeout": 7200
+      }
+    },
+    {
+      "name": "refresh-after-force-merge",
+      "operation": "refresh"
+    },
+    {
+      "name": "wait-until-merges-finish",
+      "operation": {
+        "operation-type": "index-stats",
+        "index": "_all",
+        "condition": {
+          "path": "_all.total.merges.current",
+          "expected-value": 0
+        },
+        "retry-until-success": true,
+        "include-in-reporting": false
+      }
+    },
+    {
+      "name": "knn-search-10-500",
+      "operation": "knn-search-10-500",
+      "warmup-iterations": 100,
+      "iterations": 100,
+      "clients": 4
```

Comment:
I wasn't sure what values to select here, feedback is very welcome.

Reply:
To answer this, first let's clarify if you deliberately want to run this as fast as possible; I don't see `target-throughput` specified, so it will do that. If this is the intended behavior, what value to choose here is a matter of what we want to benchmark. In this case it makes sense to keep this to `1` to avoid saturating the target.

One the other hand if you'd rather use a `target-throughput` we should run some tests on the target hardware to calculate the optimal target throughput and probably also specify a [different schedule](https://esrally.readthedocs.io/en/latest/track.html?highlight=schedule#schedule). In this case, we could also use >1 clients; in this case each client will run requests with a rate of `target throughput/clients`.

Reply:
We discussed offline and agreed it would make sense to set `clients: 1` with no specific `target-throughput`. This lets us run the track as fast as possible, avoiding unnecessary complexity. We don't have a specific reason to test request parallelism too. The rally team is thinking of moving most "standard" tracks to this simple set-up.